tests/taskTests/systemsManagerRunCommand/systemsManagerRunCommand-test.ts

```diff
@@ -132,6 +134,40 @@ describe('Systems Manager Run Command', () => {
         expect(ssm.sendCommand).toBeCalledTimes(1)
     })
 
+    test('Adds Cloudwatch output config if it exists', async () => {
+        expect.assertions(2)
+        const ssm = new SSM() as any
+        ssm.sendCommand = jest.fn(args => {
+            expect(args.CloudWatchOutputConfig.CloudWatchLogGroupName).toBe('ssm')
```

Comment:
Unsure if this test is working correctly; shouldn't this read `'cloudWatchLogGroupName'`?

If so, go ahead and add it here and we can troubleshoot later on our end. Fixing this test would be out of scope for this PR.

Reply:
The test passed, and no, it's uppercase as defined in `aws-sdk`

Reply:
Now I finally understand what you're saying. The try catch block is suspressing the errors, so the test is false positive and failed silently without me knowing. I need to temporarily remove it to see the errors `expect` would throw.